packages/indexer-agent/src/network.ts

```diff
@@ -1673,7 +1677,8 @@ export class Network {
       if (receipt === 'paused' || receipt === 'unauthorized') {
         return false
       }
-      logger.info(`Successfully claimed allocation`)
+      logger.info(`Successfully claimed allocation`, 
+        receipt.logs)
```

Comment:
Second param should be an object: `logger.info(`Successfully claimed allocation`, { logs: receipt.logs } )`